components/Header.jsx

```diff
@@ -0,0 +1,77 @@
+import Link from "next/link";
+import styled from "styled-components";
+import LibrarySvg from "@/public/library.svg";
+import ListSvg from "@/public/list.svg";
+import PlusSvg from "./PlusSvg";
+import { useRouter } from "next/router";
+
+export default function Header() {
+  const router = useRouter();
+
+  return (
+    <StyledNavigation>
+      <StyledList>
+        <li>
+          <StyledNavLink
+            href="/library"
+            aria-label="view library"
+            $active={router.pathname === "/library"}
+          >
+            <LibrarySvg />
+            <p>Library</p>
```

Comment:
It is not necessarily wrong, but it looks weird to me to have a paragraph element inside an anchor tag.

I would remove the paragraph:

```suggestion
            Library
```

It would look only minimally different